project/lolainta/README.md

```diff
@@ -0,0 +1,110 @@
+# NuScenes Software Development Kit in Cpp
+
+## Basic Information
+
+The nuScenes dataset, developed by [Motional](https://motional.com), is a large-scale public dataset for autonomous driving research. It comprises 1,000 driving scenes in Boston and Singapore, showcasing diverse and challenging scenarios. The dataset includes 1.4 million camera images, 390,000 LIDAR sweeps, 1.4 million RADAR sweeps, and 1.4 million object bounding boxes. Annotations cover 23 object classes and object-level attributes. The dataset aims to support research in computer vision and autonomous driving, emphasizing the entire sensor suite of autonomous vehicles, making it unique compared to other datasets. A supplementary dataset, nuScenes-lidarseg, offers lidar semantic segmentation. The nuScenes dataset is available for non-commercial use, including personal, educational, and research purposes. Commercial use requires contacting Motional for licensing. The dataset's purpose is to facilitate the development of safe autonomous driving technology worldwide.
```

Comment:
Please use reasonable line width (e.g., 79 chars) so that I can add comments where I want.

Reply:
done